docs/src/common_workflows/entity_embeddings/notebook.jl

```diff
@@ -0,0 +1,148 @@
+# # Entity Embeddings with MLJFlux
+
+# This demonstration is available as a Jupyter notebook or julia script
+# [here](https://github.com/FluxML/MLJFlux.jl/tree/dev/docs/src/common_workflows/entity_embeddings).
+
+# Entity embedding is newer deep learning approach for categorical encoding introduced in 2016 by Cheng Guo and Felix Berkhahn. 
+# It employs a set of embedding layers to map each categorical feature into a dense continuous vector in a similar fashion to how they are employed in NLP architectures.
+
+# In MLJFlux, the `NeuralNetworkClassifier`, `NeuralNetworkRegressor`, and the `MultitargetNeuralNetworkRegressor`` can be trained and evaluated with heterogenous data (i.e., containing categorical features) because they have a built-in entity embedding layer. 
+# Moreover, they now offer a transform which encode the categorical features with the learnt embeddings to be used by an upstream machine learning model.
```

Comment:
```suggestion
# Moreover, they now offer a `transform` method which encodes the categorical features with the learned embeddings to be used by an upstream machine learning model.
```

Reply:
I'm a little confused by what is meant by "to be used by an upstream machine learning model".

Reply:
You mean in
> Moreover, they now offer a transform which encode the categorical features with the learnt embeddings to be used by an upstream machine learning model.

I have no idea why I used the word `upstream` (and `an` should be `any`) but I basically meant that one can use the embeddings as features for any classical machine learning model.